packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -34,6 +34,16 @@ const WorkflowStepStore = types
     isThereAPreviousStep () {
       const firstStep = self.steps.keys().next()
       return self.active.stepKey !== 'summary' && self.active.stepKey !== firstStep.value
+    },
+
+    get isThereTaskHelp () {
+      const tasks = self.activeStepTasks
+
+      if (tasks.length > 0) {
+        return tasks.some(task => task.help)
```

Comment:
`Array.some` should always return false for an empty array, so this whole function could be simplified down to `return self.activeStepTasks.some(task => task.help)`